css/main.css

```diff
@@ -0,0 +1,698 @@
+
+body  {
+  width: 1300px;
+  margin: 0 auto;
+}
+
+/* CSS Off Head */
+
+header h1 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 100;
+  font-size: 300px;
+  width: 480px;
+  float: left;
+  margin-top: 0px;
+  padding: 30px 0px 300px 120px;
+  color: white;
+  line-height: .85;
+  position: relative;
+  z-index: 5;
+  letter-spacing: 1px;
+}
+
+.css_spacer {
+  letter-spacing: 27px;
+}
+
+p {
+  font-family: 'Merriweather', serif;
+}
+
+.masthead {
+  background: linear-gradient(to bottom, #cdc9dd 0%, #cdc9dd 69%, #d5d2e2 69%, #d5d2e2 100%); /* W3C */
+  width: 650px;
+  height: 1143px;
+}
+
+header p {
+  float: right;
+  margin-top: 130px;
+  color: lightgrey;
+  letter-spacing: 2.5px;
+}
+
+/* Relative Position */
+
+.position {
+  width: 925px;
+  position: relative;
+  left: 490px;
+  top: -1610px;
+}
+
+.shadow {
+  width: 600px;
+  position: absolute;
+  left: -0px;
+  bottom: 400px;
+  z-index: 1;
+}
+
+.photo {
+  width: 925px;
+  position: relative;
+  z-index: 2
+}
+
+.position h2 {
+  margin-bottom: 0px;
+  font-family: 'Merriweather', serif;
```

Comment:
duplication of font-family is unnecessary if the default font-family is set higher up the DOM element.